.github/workflows/config/default_ci_2.13.x.repos

```diff
@@ -0,0 +1,13 @@
+repositories:
+    fastcdr:
+        type: git
+        url: https://github.com/eProsima/Fast-CDR.git
+        version: v2.1.3
+    foonathan_memory_vendor:
+        type: git
+        url: https://github.com/eProsima/foonathan_memory_vendor.git
+        version: v1.3.1
+    googletest-distribution:
+        type: git
+        url: https://github.com/google/googletest.git
+        version: release-1.11.0
```

Comment:
Do we need two files, the default and the nightly? Is it possible that we want to run the jobs with different version during the night? I just ask because it will increase the maintainability

Reply:
Lets discuss it with the team